docs/HTTPClient.md

```diff
@@ -8,15 +8,15 @@ You can get a proxy calling these methods both on `Request`(the first argument o
 
 * `getServiceProxy(options)` - returns a proxy  passing through the [Microservice Gateway](https://docs.mia-platform.eu/runtime_suite/microservice-gateway/).
   * `options` - is an object with the following optional fields:
-    * `port` - an integer that identifies the port of the service to be queried.
-    * `protocol` - a string that identifies the protocol to use (only `http` and `https` are supported, the default value is `http`).
-    * `headers` - an object that represents the set of headers to send to the service;
-    * `prefix` - a string representing the prefix of the service call path.
+    * `port` - an integer that identifies the port of the service to be queried
+    * `protocol` - a string that identifies the protocol to use (only `http` and `https` are supported, the default value is `http`)
+    * `headers` - an object that represents the set of headers to send to the service
+    * `prefix` - a string representing the prefix of the service call path
     * `timeout` - set a request timeout
     * `agent` - set a custom node agent
-* `getDirectServiceProxy(serviceName, options)` - returns a direct proxy to the service.
-  * `serviceName` - The name of the service to call. You can't specify the port here, you have to do in `options.port`.
-  * `options` - The same options described above.
+* `getDirectServiceProxy(serviceNameOrURL, options)` - returns a direct proxy to the service
+  * `serviceNameOrURL` - The name of the service to call. You can pass just the hostname, without protocol and port (that you can specify into the `options` field), or a complete url string
```

Comment:
When providing a fully qualified URL how are treated port and protocol options? Are they ignored? Can we express this in the doc?

Reply:
Port and protocol are correctly used. I added this detail to the documentation